### DISCUSSION TITLE
USB NIC

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#420] [Q&A]
Hi All,

the illuminated administrator of my school bought 27 new laptop without NIC, with WiFi only!
So I'm thinking to use an USB-NIC.
But, how to boot from such NIC?

Thank you!

**Comment by alkisg** [comment]
Ask in ipxe if they support any of these USB adapters.
If not, you'll have to put at least the kernel and initrd somewhere local, e.g. disk / USB stick / SD card etc.
